bucket/xnconvert.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "1.90",
+    "description": "A fast and powerful batch image converter",
+    "homepage": "https://www.xnview.com/en/xnconvert/",
+    "license": "Freeware",
+    "suggest": {
+        "vcredist": "vcredist"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.xnview.com/XnConvert-win-x64.zip",
+            "hash": "c266e34149de6c93a44debda4746622ceaa25b1b65655658a6beaa8c95fb2f7c"
+        },
+        "32bit": {
+            "url": "https://download.xnview.com/XnConvert-win.zip",
+            "hash": "dba3bf1de553e11fac3be90cb4aba210343dc32d85f96956cce89731d970accf"
+        }
+    },
+    "extract_dir": "XnConvert",
+    "pre_install": [
+        "if (Test-Path \"$persist_dir\\xnconvert.ini\") {",
+        "    Copy-Item \"$persist_dir\\xnconvert.ini\" \"$dir\"",
+        "} else {",
+        "    ensure \"$persist_dir\" | Out-Null",
```

Comment:
Ensure in uninstaller

Reply:
I also thought it made more sense to have it in the uninstaller but didn't pay enough attention. I took it from the XnResize recipe so it might have to be changed there too.